### PR TITLE
Add resultats formulaires sur admin cours

### PR DESCRIPTION
**Pull request by ChristianOlmedo**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/View/Admin/Includes/_show_courses_inscription.html.twig [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <td>{% if pupil.experience == true %} Oui {% else %} Non {% endif %}</td>
+        <td>{{ pupil.firstname }}</td>
+        <td>{{ pupil.lastname }}</td>
+        <td>{{ pupil.email }}</td>
+        <td>{{ pupil.phone_number }}</td>
+        </tr>
+        <tr>
+    {% endfor %}
+    </tbody>
+    </table>
```
Comment (LucyMerlier) [inline src/View/Admin/Includes/_show_courses_inscription.html.twig]: indentation in table

File: src/View/Admin/Includes/_show_courses_inscription.html.twig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <td>{{ pupil.pupil_lastname }}</td>
+        <td>{{ pupil.pupil_firstname }}</td>
+        <td>{{ pupil.birthday }}</td>
+        <td>{% if pupil.experience == true %} Oui {% else %} Non {% endif %}</td>
+        <td>{{ pupil.firstname }}</td>
+        <td>{{ pupil.lastname }}</td>
+        <td>{{ pupil.email }}</td>
+        <td>{{ pupil.phone_number }}</td>
+        </tr>
+        <tr>
```
Comment (LucyMerlier) [inline src/View/Admin/Includes/_show_courses_inscription.html.twig]: lonely tr tag?

File: src/View/Admin/course_inscription.html.twig [hunk 3]
```diff
@@ -4,5 +4,9 @@
         {% include 'Admin/Includes/_navbar.html.twig' %}
     </header>
 {% endblock %}
-{% block content %}
+{% block content %} 
+    <main>
+        <h1> Administration: Formulaires d'inscription aux cours </h1>
```
Comment (LucyMerlier) [inline src/View/Admin/course_inscription.html.twig]: spaces around text? + is it really forms that we're gonna find here? or inscription sheets?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Admin/Includes/_show_courses_inscription.html.twig (hunk 1 above)
Comment (ChristianOlmedo) [reply src/View/Admin/Includes/_show_courses_inscription.html.twig]: True

File: src/View/Admin/Includes/_show_courses_inscription.html.twig (hunk 2 above)
Comment (ChristianOlmedo) [reply src/View/Admin/Includes/_show_courses_inscription.html.twig]: i admit it, poor tr tag...

File: src/View/Admin/course_inscription.html.twig (hunk 3 above)
Comment (ChristianOlmedo) [reply src/View/Admin/course_inscription.html.twig]: Done

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**